packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -904,7 +904,12 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
 ///
 /// The `useRootNavigator` argument is used to determine whether to push the
 /// popup to the [Navigator] furthest from or nearest to the given `context`. It
-/// is `false` by default.
+/// is `true` by default.
+///
+/// The `barrierColor` argument is used to determine use barrier [Color] to use.
+/// If [Color] passed is fully transparent (it's Alpha channel is equal 0) then
+/// barrier [Color] will be transparent. If `null` passed, then [Color] resolved
```

Comment:
FWIW, in Dart it isn't quite how it works. If not specified, then it'll be _kModalBarrierColor (but you can't reference a private variable in a public class doc). If you specify it to null, it will actually be null.